examples/04_tool_inference_lock/README.md

```diff
@@ -0,0 +1,9 @@
+# Tool Inference Lock
+
+This demonstrates how a tool can be given inference locking like native SketchUp tools.
+This is a continuation of "03_better_tool".
+
+* Lock tool to axes, face's planes and edge's lines with Arrow keys and Shift.
+* Using third party library for doing so.
```

Comment:
I'm not convinced we should be talking about third party libraries this early in the examples. It adds complexities I think we should avoid in this early stage of examples. It also make the example introduce two new concepts.

I'd prefer if we removed the notion of "third party library" from this example, instead convert it to a regular support file of the example.

Reply:
I've simplified it and made it just a normal source file.